packages/cli-kit/src/public/node/git.ts

```diff
@@ -268,11 +268,23 @@ export class OutsideGitDirectoryError extends AbortError {}
 export async function ensureInsideGitDirectory(directory?: string): Promise<void> {
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
-  if (!(await git({baseDir: directory}).checkIsRepo())) {
+  if (!(await insideGitDirectory(directory))) {
     throw new OutsideGitDirectoryError(`${outputToken.path(directory || cwd())} is not a Git directory`)
   }
 }
 
+/**
```

Comment:
WDYT about moving this before the theme selection step? I'd want to know about this before selecting a theme to pull

Reply:
Happy to do that, was just following the existing convention for the invalid theme directory check!

I'll extract both checks in this PR